.github/matrix.json

```diff
@@ -1 +1,60 @@
-{"config": [{"name": "Linux GCC 10","compiler": "gcc","version": "10","os": "ubuntu-20.04"},{"name": "Linux GCC 9","compiler": "gcc","version": "9","os": "ubuntu-20.04"},{"name": "Linux GCC 8","compiler": "gcc","version": "8","os": "ubuntu-20.04"},{"name": "Linux Clang 11","compiler": "clang","version": "11","os": "ubuntu-20.04"},{"name": "Linux Clang 10","compiler": "clang","version": "10","os": "ubuntu-20.04"},{"name": "Linux Clang 9","compiler": "clang","version": "9","os": "ubuntu-20.04"},{"name": "macOS apple-clang 12.0", "compiler": "apple-clang","version": "12.0","os": "macos-10.14"},{"name": "Windows VS2019","compiler": "Visual Studio","version": "16","os": "windows-2019","cmake_toolset": "Visual Studio 16 2019"},{"name": "Windows VS2017","compiler": "Visual Studio","version": "15","os": "windows-2016","cmake_toolset": "Visual Studio 15 2017 Win64"}]}
\ No newline at end of file
+{
+    "config": [
+        {
+            "name": "Linux GCC 10",
+            "compiler": "gcc",
+            "version": "10",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "Linux GCC 9",
+            "compiler": "gcc",
+            "version": "9",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "Linux GCC 8",
+            "compiler": "gcc",
+            "version": "8",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "Linux Clang 11",
+            "compiler": "clang",
+            "version": "11",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "Linux Clang 10",
+            "compiler": "clang",
+            "version": "10",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "Linux Clang 9",
+            "compiler": "clang",
+            "version": "9",
+            "os": "ubuntu-20.04"
+        },
+        {
+            "name": "macOS apple-clang 12.0",
+            "compiler": "apple-clang",
+            "version": "12.0",
+            "os": "macos-10.15"
+        },
+        {
```

Comment:
We should consider to add VS 2022.

Reply:
I tried. Conan died with a non obvious error. I think adding that will have to be a separate task.

Reply:
OK. Then later